README.md

```diff
@@ -52,3 +52,44 @@ There are three steps to the CI workflow:
      condition that a service must have been updated to run. The `deploy` job
      doesn't update any services, only manifests, so the push from the `deploy`
      job will never run another `deploy` job.
+
+## WIP: deployment to kubernetes
+
+Create a kubernetes cluster somewhere
+
+```bash
+gcloud container clusters create dapr --num-nodes=1
+```
+
+Install Dapr on the cluster (https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/)
+
+```bash
+dapr init -k
+```
+
+Install the Hello Kubernetes Quickstart example Node App. <node.yaml> is https://github.com/dapr/quickstarts/blob/master/hello-kubernetes/deploy/node.yaml. This also installs the Dapr side car, as defined in the yaml
+
+```bash
+kubectl apply -f <node.yaml>
```

Comment:
I would probably remove the angle brackets here as they're not syntactically correct and instead use the relative path to node.yaml from this directory

Reply:
actually just realised the file is somewhere else :-) But I think you can put the URL there and it'll be fine

Reply:
e.g. `kubectl apply -f https://github.com/dapr/quickstarts/blob/master/hello-kubernetes/deploy/node.yaml`